pkg/comp-functions/functions/vshnnextcloud/files/install-collabora.sh

```diff
@@ -0,0 +1,37 @@
+ /bin/bash -x
+
+IS_OPENSHIFT=$1
+NAMESPACE=$2
+NEXTCLOUD_INSTANCE=$3
+COLLABORA_FQDN=$4
+COLLABORA_URL="https://$COLLABORA_FQDN"
+
+
+/usr/bin/kubectl wait --for=condition=available --timeout=600s -n "$NAMESPACE" "deployments/$NEXTCLOUD_INSTANCE"
+
+if [ "$IS_OPENSHIFT" == "true" ]; then
```

Comment:
Nit: a lot of duplicated code that takes more time to review than necessary

Reply:
what do You mean, where is duplicated code?

Reply:
between if and else clause the commands and logic is basically the same.

Reply:
`almost` is the keyword here, in kindev You need to switch user to run commands without issues, pretty painful to refactor and test everywhere

Reply:
This was a nitpick, so it's ok for now as is.